canonical-web-and-design/maas-ui
# Handle server down errors

## Done
- Handle errors when checking if the user is authenticated and the server is down.

## QA
- Open proxy/.env.local and change MAAS_URL to a URL/IP that DOESN'T exist.
- Load the UI.
- You should see a message about not being able to access the server.
- Open proxy/.env.local and change MAAS_URL to URL/IP that DOES exist.
- Load the UI.
- You should see the login page or a logged in page (if you were previously logged in).

## Fixes
Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/910.
